src/main/java/org/opensearch/neuralsearch/ml/MLCommonsClientAccessor.java

```diff
@@ -311,4 +310,18 @@ private MLInput createMLMultimodalInput(final List<String> targetResponseFilters
         final MLInputDataset inputDataset = new TextDocsInputDataSet(inputText, modelResultFilter);
         return new MLInput(FunctionName.TEXT_EMBEDDING, null, inputDataset);
     }
+
+    private boolean checkIfStringIsNotBlank(String value) {
```

Comment:
You can use [this function](https://github.com/opensearch-project/OpenSearch/blob/main/libs/core/src/main/java/org/opensearch/core/common/Strings.java#L131-L173) from opensearch core to achieve this functionality.

We need to replace `org.apache.logging.log4j.util.Strings` with `org.opensearch.core.common.Strings`

Reply:
Also, you might need to raise a separate PR to fix it in main branch